src/openfluid/ui/waresdev/WareSrcActionCollection.cpp

```diff
@@ -0,0 +1,225 @@
+  m_Actions["SaveAsFile"]->setToolTip(tr("Save the current file as..."));
+  m_Actions["SaveAsFile"]->setEnabled(false);
+
+  m_Actions["SaveAllFiles"] = new QAction(openfluid::ui::common::getIcon("file-save","/ui/common",false),
+                                          tr("Save all"), this);
+  if (m_WithShortcuts)
+  {
+    m_Actions["SaveAllFiles"]->setShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_S));
+  }
+  m_Actions["SaveAllFiles"]->setToolTip(tr("Save all files of the ware"));
+  m_Actions["SaveAllFiles"]->setEnabled(false);
+
+  m_Actions["CloseFile"] = new QAction(openfluid::ui::common::getIcon("close","/ui/common",false),
+                                       tr("Close"), this);
+  m_Actions["CloseFile"]->setToolTip(tr("Close the current file"));
+  m_Actions["DeleteFile"] = new QAction(tr("Delete"), this);
+  m_Actions["DeleteFile"]->setToolTip(tr("Delete the current file"));
+
+
+  // ====== Edit ======
+
+  m_Actions["Copy"] = new QAction(tr("Copy"), this);
+  if (m_WithShortcuts)
+  {
+    m_Actions["Copy"]->setShortcuts(QKeySequence::Copy);
+  }
+
+  m_Actions["Cut"] = new QAction(tr("Cut"), this);
+  if (m_WithShortcuts)
+  {
+    m_Actions["Cut"]->setShortcuts(QKeySequence::Cut);
+  }
+
+  m_Actions["Paste"] = new QAction(tr("Paste"), this);
+  if (m_WithShortcuts)
+  {
+    m_Actions["Paste"]->setShortcuts(QKeySequence::Paste);
+  }
+
+  m_Actions["FindReplace"] = new QAction(openfluid::ui::common::getIcon("search","/ui/common",false), 
+                                         tr("Find/Replace"), this);
+  if (m_WithShortcuts)
+  {
+    m_Actions["FindReplace"]->setShortcuts(QKeySequence::Find);
+  }
+
+  m_Actions["GoToLine"] = new QAction(tr("Go to line..."), this);
+  if (m_WithShortcuts)
+  {
+    m_Actions["GoToLine"]->setShortcut(QKeySequence(Qt::CTRL + Qt::Key_L));
+  }
+
+
+  // ====== Build ======
+
+  // Configure ware
+
+  m_Actions["ConfigureWare"] = new QAction(openfluid::ui::common::getIcon("configure","/ui/common",false),
+                                           tr("Configure"), this);
+  if (m_WithShortcuts)
+  {
+    m_Actions["ConfigureWare"]->setShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_B));
+  }
+
+
+  // Build ware
+
+  m_Actions["BuildWare"] = new QAction(openfluid::ui::common::getIcon("build","/ui/common",false),
+                                       tr("Build"), this);
+  if (m_WithShortcuts)
+  {
+    m_Actions["BuildWare"]->setShortcut(QKeySequence(Qt::CTRL + Qt::Key_B));
+  }
+
+
+  // Generate doc
+
+#if OPENFLUID_SIM2DOC_ENABLED
+    m_Actions["GenerateDoc"] = new QAction(openfluid::ui::common::getIcon("generate-doc","/ui/common",false),
+                                          tr("Generate doc"), this);
+    if (m_WithShortcuts)
+    {
+      m_Actions["GenerateDoc"]->setShortcut(QKeySequence(Qt::CTRL + Qt::Key_G));
+    }
+#endif
+
+
+  // Options
+
+  m_Actions["WareOptionsMenu"] = new QAction(openfluid::ui::common::getIcon("configure","/ui/common",!m_IsIncluded),
```

Comment:
Are these actions for the DevStudio statusbar?

Where is displayed/used these texts?
Same as in the option dialog?

Reply:
Seems to be deprecated, to be cleaned